core/src/main/java/com/onewelcome/core/usecase/GetMobileAuthenticationEnrollmentStatusUseCase.kt

```diff
@@ -0,0 +1,29 @@
+package com.onewelcome.core.usecase
+
+import com.github.michaelbull.result.Result
+import com.github.michaelbull.result.map
+import com.onegini.mobile.sdk.android.model.entity.UserProfile
+import com.onewelcome.core.entity.MobileAuthEnrollmentStatus
+import com.onewelcome.core.omisdk.facade.OmiSdkFacade
+import javax.inject.Inject
+
+class GetMobileAuthenticationEnrollmentStatusUseCase @Inject constructor(
+  private val omiSdkFacade: OmiSdkFacade,
+  private val getUserProfilesUseCase: GetUserProfilesUseCase
```

Comment:
I thought we agreed to keep useCases separated, without dependencies on other useCases. With this concept, execute should get userProfiles as a parameter.